packages/marketplace/src/selector/developer-detail.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReduxState } from '@/types/core'
```

Comment:
`packages/marketplace/src/selector/developer-detail.tsx` to

`packages/marketplace/src/selector/developer.ts`